src/main/java/utils/RandomGenerator.java

```diff
@@ -17,7 +17,7 @@ public static int randomNumber() {
 
     public static String randomName() {
         String name = UUID.randomUUID().toString();
-        return name.replaceAll("-", "");
+        return name.replaceAll("-", "").substring(5);
     }
```

Comment:
do we need to make this change? wouldnt it be better to create a real string generator? what we have here is an UUID generator that deletes the "-"

Reply:
I made this change in order to obtain a string which is shorter than the usual 32 char UUID. It seemed simpler than creating a string generator.

Reply:
if we refactor the code, we will have to rethink this. its not ok to generate an UUID and remove - when we can create a string instead